### PR TITLE
Small edits to Lambdas.

### PR DESCRIPTION
**Pull request by JoshMcCullough**

### CODE REVIEW

File: lambdas.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ normal functions and even behave like them.
 
 .. code:: python
 
-    lambda argument: manipulate(argument)
+    lambda arguments: expression
```
Comment (yasoob) [inline lambdas.rst]: IMO `manipulate(argument)` seems like a better choice. Perhaps you could revert this change and just go with the *ways* edit?

Comment (JoshMcCullough) [inline lambdas.rst]: I was trying to illustrate that you can pass multiple arguments. This is how the Python docs do it (although honestly I didn't confirm it until now). Happy to revert if you still don't agree. After all, it is your book! :)
